source/read/distinct.txt

```diff
@@ -0,0 +1,173 @@
+free MongoDB Atlas cluster and load the sample datasets, see
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+``distinct()`` Method
+---------------------
+
+To retrieve the distinct values for a specified field, call the ``distinct()``
+method and pass in the name of the field you want to find distinct values for.
+
+Retrieve Distinct Values Across a Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example retrieves the distinct values of the ``borough`` field in
+the ``restaurants`` collection:
+
+.. io-code-block::
+
+   .. input:: /includes/read-ops/distinct.java
+      :start-after: start-distinct
+      :end-before: end-distinct
+      :language: java
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Bronx
+      Brooklyn
+      Manhattan
+      Missing
+      Queens
+      Staten Island
+
+The results show every distinct value that appears in the ``borough`` field
+across all documents in the collection. Although several documents have the same
+value in the ``borough`` field, each value appears in the results only once.
+
+Retrieve Distinct Values Across Specified Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can provide a **query filter** to the ``distinct()`` method to find the distinct
+field values across a subset of documents in a collection. A query filter is an expression that specifies search
+criteria used to match documents in an operation. For more information about
+creating a query filter, see :ref:`java-rs-specify-query`.
+
+The following example retrieves the distinct values of the ``borough`` field for
+all documents that have a ``cuisine`` field value of ``"Italian"``:
+
+.. io-code-block::
+
+   .. input:: /includes/read-ops/distinct.java
+      :start-after: start-distinct-query
+      :end-before: end-distinct-query
+      :language: java
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Bronx
+      Brooklyn
+      Manhattan
+      Queens
+      Staten Island
+
+Modify Distinct Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``distinct()`` method can be modified by chaining methods to the ``distinct()`` method
+call. If you don't specify any options, the driver does not customize the operation.
+
+The following table describes some methods you can use to customize the
+``distinct()`` operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``batchSize()``
+     - | Sets the number of documents to return per batch.
+
+   * - ``collation()``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``comment()``
+     - | Specifies a comment to attach to the operation.
+
+   * - ``filter()``
+     - | Sets the query filter to apply to the query.
```

Comment:
Q: Does this mean that you can either provide the filter as a parameter, or chain this method and it will do the same thing? Or does it do something different? 
If it has the same output as providing it in the parameter then I don't think we need to change anything, but if it's different we should note that here

Reply:
The result is the same, but you raise a good point – let me adjust the code sample to use the `.filter()` method instead of the `distinct()` method that takes the filter as the parameter, since I personally think it's more readable.